src/Resources/config/makers.xml

```diff
@@ -51,6 +51,11 @@
                 <tag name="maker.command" />
             </service>
 
+            <service id="maker.maker.make_event" class="Symfony\Bundle\MakerBundle\Maker\MakeEvent">
```

Comment:
I think we have to keep the `maker.maker.make_subscriber` to avoid breaking changes if it a service decorates it.

Reply:
Revert deletion and add deprecation